.github/workflows/codequality.yml

```diff
@@ -1,4 +1,4 @@
-name: Code quality - lint, typecheck, tests
+name: Code quality - lint, typecheck, unit tests
```

Comment:
Kinda want to split this fiel up into 3 single ones to make it easier to see what exactly fails on a PR. What do you think?

Reply:
Can be done via #912